editor/src/clj/editor/gui.clj

```diff
@@ -525,7 +525,7 @@
 
 ;; Base nodes
 
-(def base-display-order [:id :generated-id scene/SceneNode :manual-size])
+(def base-display-order [:id :generated-id scene/SceneNode])
```

Comment:
The `:manual-size` property is only supposed to be visible on `TextNode`, and it already lists it in its `(display-order ...)` declaration.